infra/ansible/README.md

```diff
@@ -22,4 +26,95 @@ Deploy the Batcher:
 
 ```shell
 make ansible_batcher_deploy INVENTORY=</path/to/inventory> KEYSTORE=<path/to/keystore>
-```
\ No newline at end of file
+```
+
+## Operator
+
+> [!IMPORTANT]
+> You need to have previously created an ECDSA keystore with at least 1ETH and a BLS keystore.
+> You can create keystore following this [guide](#How-to-Create-Keystores)
+
+> [!CAUTION]
+> To register the Operator in Aligned successfully, you need to have been whitelisted by the Aligned team previously.
+
+
+To deploy the Operator you need to set some variables and then run the Operator playbook.
+
+Create the variables files:
+
+```shell
+make ansible_operator_create_env
+```
+
+This will create the following files in `infra/ansible/playbooks/ini`:
+
+- `config-operator.ini`
+- `config-register-operator.ini`
+
+Deploy the Operator:
+
+```shell
+make ansible_operator_deploy INVENTORY=</path/to/inventory> ECDSA_KEYSTORE=</path/to/ecdsa/keystore> BLS_KEYSTORE=</path/to/bls/keystore>
+```
+
+# How to Create Keystores
+
+## Create ECDSA Keystore
+
+Make sure you have installed:
+
+- [Foundry](https://book.getfoundry.sh/getting-started/installation)
+
+Now you can create the ECDSA keystore using the following command:
+
+```shell
+cast wallet new .
```

Comment:
I would put the eigen command here to create the wallet as the `eigenlayercli` expects for the eigenlayer type of keystore. That or add the command to import `eigenlayer` from an existing privatekey: 

```shell
 eigenlayer operator keys import --key-type <TYPE> <KEY_NAME> <PRIV_KEY>
```

Reply:
Done